src/lib/constant_mappings.cpp

```diff
@@ -139,4 +142,16 @@ const boost::bimap<DataType, std::string> data_type_to_string =
                  return map;
                });
 
+const boost::bimap<EncodingType, std::string> encoding_type_to_string = make_bimap<EncodingType, std::string>({
+    {EncodingType::DeprecatedDictionary, "Dictionary (Deprecated)"},
+    {EncodingType::Dictionary, "Dictionary"},
+    {EncodingType::RunLength, "RunLength"},
+});
+
+const boost::bimap<VectorCompressionType, std::string> vector_compression_type_to_string =
```

Comment:
What do we need the bimap for? If, at some point, we need to convert "Fixed-size byte-aligned" (but not "fixed-size byte-aligned" or "fixed size byte aligned") to a `VectorCompressionType`, we probably did something wrong.